pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -159,6 +159,8 @@ var _ = Describe("Pod Network", func() {
 
 		//For Bridge tests
 		mockNetwork.EXPECT().LinkByName(podInterface).Return(dummy, nil)
+		mockNetwork.EXPECT().AddrList(dummy, netlink.FAMILY_ALL).Return(addrList, nil)
+		mockNetwork.EXPECT().LinkByName(podInterface).Return(dummy, nil)
```

Comment:
Duplicates line 161

Reply:
I see, can you please use mockNetwork.EXPECT().LinkByName(podInterface).Return(dummy, nil).Times(2) instead of duplicating the line?